test/cpp/api/modules.cpp

```diff
@@ -532,6 +558,14 @@ TEST_F(ModulesTest, PrettyPrintEmbedding) {
       "torch::nn::Embedding(count=10, dimension=2)");
 }
 
+TEST_F(ModulesTest, PrettyPrintCosineSimilarity) {
+  ASSERT_EQ(c10::str(CosineSimilarity()), "torch::nn::CosineSimilarity()");
```

Comment:
We should change the expected string after we change `pretty_print` of the new modules.